tests/integration-tests/configs/slurm_accounting.yaml

```diff
@@ -0,0 +1,20 @@
+{%- import 'common.jinja2' as common -%}
+---
+test-suites:
+  schedulers:
+    test_slurm_accounting.py::test_slurm_accounting:
+      dimensions:
+        - regions: ["us-east-1"]
+          instances: ["c5.xlarge", "c6g.xlarge"]
+          oss: ["alinux2", "ubuntu2004"]
+#          instances: ["c5.xlarge"]
+#          oss: ["alinux2"]
+          schedulers: ["slurm"]
+    test_slurm_accounting.py::test_slurm_accounting_disabled_to_enabled_update:
+      dimensions:
+        - regions: ["us-east-1"]
+          instances: ["c5.xlarge", "c6g.xlarge"]
+          oss: ["alinux2", "ubuntu2004"]
+#          instances: ["c5.xlarge"]
+#          oss: ["alinux2"]
+          schedulers: ["slurm"]
```

Comment:
If you want to run these integration tests daily, I believe you may still have to add them to [`/tests/integration-tests/configs/common/common.yaml`](https://github.com/aws/aws-parallelcluster/blob/develop/tests/integration-tests/configs/common/common.yaml).

Reply:
Yes, thank you for the reminder.

Reply:
Tests moved to common.yaml.